Dockerfile.hip

```diff
@@ -9,12 +9,15 @@ FROM $BASE_IMAGE
 
 ENV DEBIAN_FRONTEND=noninteractive
 
+# Install required software for rocm installation
+RUN apt-get install -y wget gnupg
+
```

Comment:
Can we add this to the `RUN` step below instead? Possibly with an `apt-get update` before it?

Reply:
I was just referring to the separate `RUN` step, installing ´wget` and `gnupg` is all good.

Regarding the update, we remove all the temporary apt files after we install everything in the base image, so the update is necessary.